tasks.py

```diff
@@ -41,6 +41,14 @@ def deploy(h: DeployHost) -> None:
     g.run_function(deploy)
 
 
+@task
+def sotp(c: Any, acct: str) -> None:
+    """
+    Get TOTP token from sops
+    """
+    c.run(f"nix develop .#sotp -c sotp {acct}")
```

Comment:
I think you could get away with making this nix runnable no? `f"nix run .#sotp {acct}"` (and change the above recipe)

Reply:
Once this package is added to nixpkgs we'd be changing it back to a devshell anyway?